src/index.tsx

```diff
@@ -1,13 +1,30 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PageNotFound } from './components/PageNotFound';
+import { PeoplePage } from './components/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
   <Router>
-    <App />
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element=<Navigate to="/" /> />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slug?" element={<PeoplePage />} />
```

Comment:
The nested route `<Route path=":slug?" element={<PeoplePage />} />` inside the `people` route might cause the `PeoplePage` component to render twice when a `slug` is present. Consider reviewing the routing logic to ensure the component is rendered as intended.